### PR TITLE
Add follow-on fields to desktop close work order form

### PR DESCRIPTION
**Pull request by LBHCallumM**
## Ticket

[https://hackney.atlassian.net/browse/HPT-375?atlOrigin=eyJpIjoiY2Y2NmJjNzZhNjFkNDc5MDhjYzVkMjg0MzgzZjZiODQiLCJwIjoiaiJ9](https://hackney.atlassian.net/browse/HPT-375?atlOrigin=eyJpIjoiY2Y2NmJjNzZhNjFkNDc5MDhjYzVkMjg0MzgzZjZiODQiLCJwIjoiaiJ9)

## Summary of Changes

Add follow-on fields to desktop close work order form. 

- Extract fields into separate components, so they can be reused for each form
- Update summary page for desktop form (and refactor)
- Re-enable pre-existing tests that had been disabled
- Update snapshots

### Follow-on reason
image

### Follow-on fields visible
image

### CODE REVIEW

**Review by adamtry — APPROVED** [review]
Well organised, looks good

File: src/components/WorkOrders/CloseWorkOrderByProxy.js [hunk 1]
```diff
@@ -169,6 +190,32 @@ const CloseWorkOrderByProxy = ({ reference }) => {
     )
     setCompletionDate(formattedCompletionDate)
 
+    // set follow on data
+
+    if (formData['followOnStatus'] === 'furtherWorkRequired') {
+      const requiredFollowOnTrades = []
+
+      FOLLOW_ON_REQUEST_AVAILABLE_TRADES.forEach((x) => {
+        console.log({ x })
```
Comment (Duslerke) [inline src/components/WorkOrders/CloseWorkOrderByProxy.js:199]: probs not needed on prod.

**Review by Duslerke — COMMENTED** [review]
left a comment :D

File: src/components/WorkOrders/CloseWorkOrderByProxy.js (hunk 1 above)
Comment (LBHCallumM) [reply src/components/WorkOrders/CloseWorkOrderByProxy.js:199]: no

**Pull request merged**